static/js/things.js

```diff
@@ -67,6 +67,11 @@ var ThingsScreen = {
               // eslint-disable-next-line no-unused-vars
               var newOnOffSwitch = new OnOffSwitch(description);
               break;
+            case 'colorLight':
```

Comment:
This isn't one of the [Web Thing Types](https://iot.mozilla.org/wot/#dimmablecolorlight-type) defined so far. Do you think we need an onOffColorLight type as well?